### DISCUSSION TITLE
Sveltekit + Tailwindcss JS not working

### DISCUSSION THREAD

**Discussion by havullc** [tailwindlabs/tailwindcss#9115] [Help]
JS is not working !!
Please help me somebody!!

I install "Sveltekit + Tailwindcss" to my local according from bellow url.
https://tailwindcss.com/docs/guides/sveltekit

Then copy the code from "Centered with bottom border"  from bellow url.
https://tailwindui.com/components/preview#component-1c0390a81b16cf5028aeff08be955766

**Comment by wongjn** [comment]
Hi there, what about is it about the JS is not working? Which bit of JS, browser JavaScript or Node.js JavaScript? Are there any errors? A more detailed explanation of what isn't working and code of your project would be helpful!
